spec/ace.md

```diff
@@ -27,12 +27,28 @@ Each app in a pod will start chrooted into its own unique read-write filesystem
 
 An app's filesystem must be *rendered* in an empty directory by the following process (or equivalent):
 
-1. If the ACI contains a non-empty `dependencies` field in its `ImageManifest`, the `rootfs` of each dependent image is extracted into the target directory, in the order in which they are listed.
-2. The `rootfs` contained in the ACI is extracted into the target directory
-3. If the ACI contains a non-empty `pathWhitelist` field in its `ImageManifest`, *all* paths not in the whitelist must be removed from the target directory
+1. If the ACI contains a non-empty `dependencies` field in its `ImageManifest`, the `rootfs` of each dependent image is extracted into the target directory, in the order in which they are listed. The multiple dependencies can form a tree. The dependencies are resolved in the depth-first order.
```

Comment:
Actually, if we allow dependencies like:
```
A -> [B, C]
B -> [D]
C -> [D]
```

then it is not a tree anymore, but rather a directed acyclic graph, right?